cloudca/resource_cloudca.go

```diff
@@ -53,7 +53,7 @@ func readIntFromString(valStr string) int {
 func handleNotFoundError(err error, d *schema.ResourceData) error {
 	if ccaError, ok := err.(api.CcaErrorResponse); ok {
 		if ccaError.StatusCode == 404 {
-			fmt.Errorf("Entity (id=%s) not found", d.Id())
+			_ = fmt.Errorf("Entity (id=%s) not found", d.Id())
```

Comment:
Sorry if I wasn't clear enough before. Lines like these are useless. It creates a string then it is ignored. It doesn't log it or anything. We should either remove the lines or log them at debug level.

Reply:
In 20e3b98 I reverted and applied error handling for all the rest, but these ones were never actually logged and printed before. That's why I didn't touch them.
I'm going to bubble these up too.

Reply:
Ok cool. Be careful though, in some cases a not found may be acceptable (e.g. delete instance, but instance is not found)

Reply:
83c5b00 unifies this behavior across all files.